tests/utils/test_utils.go

```diff
@@ -52,37 +55,23 @@ func init() {
 		}
 		testLogger.Println("Test Starting:")
 	}
-}
 
-// Called from a test program.
-//  - determines the test program name
-// 	- creates a temproray directory for the test program
-// returns the name of the directory created.
-func GetTempDir() string {
-	_, fn, _, ok := runtime.Caller(1)
-	if !ok {
-		return tmpDir
-	}
-	testFile := filepath.Base(fn)
-	testFileExtension := filepath.Ext(testFile)
-	subdir := testFile[0 : len(testFile)-len(testFileExtension)]
-	return CreateTempDir(subdir)
+	testError(true)
+	testError1(true)
```

Comment:
Are these func and their calls supposed to be removed?

Reply:
oops.